### ISSUE TITLE
puppet provider uses @config.expanded_manifests_path which no longer exists in vagrant 1.4.1

### ISSUE THREAD

**Issue by wscheele** [WinRb/vagrant-windows#158]
starting a vagrant box (with vagrant 1.4.1) that uses the vagrant-windows plugin (1.5.0) and the puppet provisioner produces the following error:

```
/opt/vagrant/embedded/gems/gems/vagrant-1.4.1/lib/vagrant/action/builtin/synced_folders.rb:41:in `expand_path': can't convert Vagrant::Config::V2::DummyConfig to String (Vagrant::Config::V2::DummyConfig#to_str gives Vagrant::Con
fig::V2::DummyConfig) (TypeError)
```

the error is caused by line 97 of:

```
~/.vagrant.d/gems/gems/vagrant-windows-1.5.0/lib/vagrant-windows/monkey_patches/plugins/provisioners/puppet/provisioner/puppet.rb
```

which calls the expanded_manifests_path on the vagrant puppet config. this method no longer exists.

i've replaced line 97 with an inlined version of the old implementation as a workaround:

``` ruby
@expanded_manifests_path = Pathname.new(@config.manifests_path[1]).expand_path(root_path)
```

**Comment by sneal** [comment]
Thanks for reporting this

**Comment by alexbhr** [comment]
:+1: on fixing that, I hit it too. Would you accept a PR with the above fix?

**Comment by sneal** [comment]
@alexbhr Absolutely. If you provide the PR, I'll cut a new gem.

**Comment by alexbhr** [comment]
Would this fix be acceptable, or do you recommend going about this in a different way?

**Comment by sneal** [comment]
I think I might have a cleaner fix that removes that entire monkey patched method.

**Comment by sneal** [comment]
Fixed in 1.5.1
